flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -275,8 +275,28 @@
 	 *
 	 * @param table The table for which the AST and execution plan will be returned.
 	 */
+	@Deprecated
```

Comment:
Why is it deprecated? I think it is valid. I think we can safely assume that by default `extended = false`.

Reply:
ok, i will revert it. i want to unify `public String explain(Table table)` into `public String explain(Table table, boolean extended)` before, which looks more simple.